### ISSUE TITLE
Labels for select_ fields are grey, black for other fields

### ISSUE THREAD

**Issue by michaelkohn** [medic/config-covid#48]
For some of the `select_` fields, the question (label) text is grey. For other fields, the question (label) is black.  I think this is a bug but want to confirm.  If it is a bug, please raise in medic/cht-core

![Screen Shot 2020-04-08 at 12 24 35 AM](https://user-images.githubusercontent.com/10096985/78744417-816c3e80-792f-11ea-9f8f-1da52fd9e78e.png)

**Comment by abbyad** [comment]
Yes, this is a bug and should be opened in cht-core. I thought we had it open there already, but I can't find it.
